binderhub/tests/test_registry.py

```diff
@@ -69,7 +69,7 @@ def initialize(self, test_handle):
         self.test_handle = test_handle
 
     def get(self):
-        scope = self.get_argument("scope")
+        _ = self.get_argument("scope")
```

Comment:
Since the result isn't needed maybe the whole line can be deleted? Not a big deal to keep it though.